src/Microsoft.TestPlatform.CrossPlatEngine/Client/InProcessProxyDiscoveryManager.cs

```diff
@@ -95,6 +95,12 @@ public void Abort()
         Task.Run(() => _testHostManagerFactory.GetDiscoveryManager().Abort());
     }
 
+    /// <inheritdoc/>
+    public void Abort(ITestDiscoveryEventsHandler2 eventHandler)
```

Comment:
Shall we make this async?

Reply:
We have a parameterless `Abort` overload that returns `void` so maybe this one should be the same. I don't know whether we prefer uniform APIs or try to have new ones differnet.

Reply:
> But we don't want to make such change now, right?

true.

Reply:
Closing this thread as we won't do any change in this PR.